test/libsolidity/syntaxTests/parsing/emit_event.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+	event A();
+	function f() {
+		emit A();
```

Comment:
I'm sure there's a test like this already?

Reply:
@axic We have some syntax tests that cover the usage of ``emit`` with non-event types, but none that doesn't expect parser / type errors. So I think this should be added.

Reply:
@axic Oh, you're right. I forgot to search in ``SolidityNameAndTypeResolution``. Will remove this one then.